js/drawobjects/circle.ts

```diff
@@ -25,7 +25,7 @@ export class Circle implements DrawObject {
     return Math.abs(distance(this.center, point) - this.radius);
   }
 
-  getErasingObjects(eraserCenter: Point, eraserRadius: number): DrawObject[] {
+  getObjectsToReplaceWithWhenErasing(eraserCenter: Point, eraserRadius: number): DrawObject[] {
```

Comment:
Think we've gone too far in the other direction, might be a tad bit too verbose. Perhaps `splitWithEraser` and clarify what exactly this means in a JSDoc comment?

Reply:
Why is verbosity bad for this method? The method name is currently written only 4 times in the entire code base.

Reply:
🤷‍♂️ Guess it doesn't matter all that much.